### ISSUE TITLE
Adding public information about CI status of the library

### ISSUE THREAD

**Issue by elhigu** [oracle/node-oracledb#848]
I could write Travis CI configuration which runs driver tests with node 4,6,8 and 9 against Oracle Express g11 and add build status badge. Would you like PR for that?

**Comment by cjbj** [comment]
@octokyle runs all those tests (and many more, eg. with different Oracle Clients and DBs) during our release cycle so I think you can save time and work on other enhancements!  But the offer is appreciated.

**Comment by elhigu** [comment]
Well CI should have catched for example https://github.com/oracle/node-oracledb/issues/847 already before deploy process when error landed to master (well maybe would have, I don't know if that part was covered by tests suite). 

Automatic testing is usually more robust and catches them earlier than running tests manually. Of course CI doesn't replace separate deploy testing completely, since CI cannot test against all oracle db versions etc.

Offer is open though. I don't think I want to be involved with other enchantments though, I have already my hands full with knex maintenance :) That travis CI setup would just have been pretty easy to copy-paste from my other repos.

**Comment by cjbj** [comment]
@elhigu let's not mention #847 until we know what that happened there !!!

Have you got a CI solution that satisfies all Oracle license click-through requirements?

**Comment by elhigu** [comment]
@cjbj sure, lets not mention that :) I actually verified that our approach was fine with people from local oracle representation which they confirmed that it is all good. 

It sounds pretty reasonable that they actually want to enable use of oracle dbs to be more accessible for people in node community (we would need to drop the support right away if we couldn't run CI on it).

**Comment by cjbj** [comment]
What's valid for you may not be for me - or may not be easy. Excuse me getting grumpy at the thought having to sort out legals and business stuff. Particularly for things that seem no-brainers to mere mortals like us.

**Comment by elhigu** [comment]
I see you are worried about it making life harder if this feature would be merged and that sounds perfectly good reason to me.

> Would you like PR for that?

tl;dr answer was no :)

**Comment by cjbj** [comment]
@elhigu thanks for understanding.  Since we push commits to an internal repo first, the best place to do CI is there, and not duplicate effort etc on GitHub.  There are more than enough other projects to do.
